normandy/recipes/tests/test_serializers.py

```diff
@@ -28,6 +31,46 @@ def test_it_works(self, rf):
             'is_approved': recipe.is_approved
         }
 
+    def test_it_validates_arguments(self):
+        schema = json.loads(
+            open('normandy/recipes/static/actions/console-log/package.json').read()
```

Comment:
It is probably fine to use this schema file directly (or the heartbeat one). Mocking out an entirely fake one has the advantage of being able to get exactly the data you need to test the behavior you want.